packages/xforms-engine/src/parse/body/BodyDefinition.ts

```diff
@@ -48,7 +48,7 @@ const BodyElementDefinitionConstructors = [
 	PresentationGroupDefinition,
 	StructuralGroupDefinition,
 	InputControlDefinition,
-	SelectDefinition,
+	SelectControlDefinition,
```

Comment:
Maybe this will expand to something like `ItemBasedControlDefinition` or something with the addition of `rank`?

Reply:
I can see that becoming an abstract base class for select/select1/rank. Both concrete classes would still be enumerated here though, as it's used for instantiation during parsing. And the base class would make even more sense when we split select/select1.